test/fixtures/privileges.sql

```diff
@@ -62,9 +62,12 @@ GRANT USAGE ON SEQUENCE
 TO postgrest_test_anonymous;
 
 -- Privileges for non anonymous users
+GRANT USAGE ON SCHEMA test TO postgrest_test_default_role;
 GRANT USAGE ON SCHEMA test TO postgrest_test_author;
 GRANT ALL ON TABLE authors_only TO postgrest_test_author;
 
 GRANT SELECT (article_id, user_id) ON TABLE limited_article_stars TO postgrest_test_anonymous;
 GRANT INSERT (article_id, user_id) ON TABLE limited_article_stars TO postgrest_test_anonymous;
 GRANT UPDATE (article_id, user_id) ON TABLE limited_article_stars TO postgrest_test_anonymous;
+
+ALTER ROLE postgrest_test_authenticator set search_path = 'public', 'test';
```

Comment:
I fixed that dependency on the ```search_path``` on some of the tests just by schema qualifying some of the definitions on ```fixtures/schema.sql``` but there were some others tests related to computed columns that didn't work, so I leaved that as it is to maybe fix it in another issue.